src/api-reference/expense/expense-report/v4.comments.markdown

```diff
@@ -235,6 +235,240 @@ curl --location --request GET 'https://us.api.concursolutions.com/expensereports
     }
 ]
 ```
+## <a name="post-report-header-comments"></a>Post Report Header Comments
+
+Creates a new comment for the specified report.
+
+### Scopes
+
+`expense.report.read` - Refer to [Scope Usage](#scope-usage) for full details.
+
+### Request
+
+#### URI Template
+
+```shell
+https://{datacenterURI}/expensereports/v4/users/{userID}/context/{contextType}/reports/{reportId}/comments
+```
+
+#### Parameters
+
+|Name|Type|Format|Description|
+|---|---|---|---|
+|`userID`|`string`|-|**Required** The unique identifier of the SAP Concur user. Use [Identity v4](/api-reference/profile/v4.identity.html) to retrieve the `userID`.|
+|`contextType`|`string`|-|**Required** The access level of the SAP Concur user, which determines the form fields they can view/modify. Supported values: `TRAVELER`, `PROXY`|
+|`reportId`|`string`|-|**Required** The unique identifier of the report that is being read.|
+
```

Comment:
We have a optional parameter `accept-language`.  Please check that we should add that too or not.